compass-js-frontend/src/main/webapp/collection/scenenode-collection.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * This file is part of COMPASS. It is subject to the license terms in
+ * the LICENSE file found in the top-level directory of this distribution.
+ * (Also available at http://www.apache.org/licenses/LICENSE-2.0.txt)
+ * You may not use this file except in compliance with the License.
+ */
+
+"use strict";
+
+var app = require('ampersand-app');
+var CompassCollection = require("./compass-collection");
+var Config = require("../config");
+
+var SceneNodeCollection = CompassCollection.extend({
+	url: Config.getRESTPath("scenenodes/")
+	//we do not specifiy the model (SceneNode) in order to avoid a circular dependency
```

Comment:
can we specify this via the constructor? s.t. the SceneNode object reference remains local to scenenode.js, but this collection still knows its model constructor; thus avoiding circular `require()`